src/renderer/components/app.tsx

```diff
@@ -18,7 +18,7 @@
  * IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
  * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
  */
-
+import "tailwindcss/tailwind.css";
```

Comment:
Why not imported into `app.scss`?

Reply:
Moved into `app.scss` but it requires 3 lines instead of 1:
```
@import "tailwindcss/base";
@import "tailwindcss/components";
@import "tailwindcss/utilities";
@import "~flex.box";
@import "fonts";
```

Reply:
Why? You can import `.css` intro `.scss` afaik.. (just explicity add file extension)